src/provable/from-layout.ts

```diff
@@ -409,66 +389,6 @@ function ProvableFromLayout<
     );
   }
 
-  type FoldSpec<T, R> = {
```

Comment:
this code was just moved out of a function to use it elsewhere, but not really changed